CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+# Copyright (c) 2019, QuantStack and Mamba Contributors
```

Comment:
Should LICENSE.md be updated to have the same copyright date?

Reply:
Fixed. Good catch.